Libraries/LibGUI/FilePicker.h

```diff
@@ -74,7 +74,7 @@ class FilePicker final : public Dialog {
     RefPtr<Label> m_preview_image_label;
     RefPtr<Label> m_preview_name_label;
     RefPtr<Label> m_preview_geometry_label;
-    Mode m_mode { Mode::Open };
+    Mode m_mode;
```

Comment:
Let's always prefer initialization at the declaration instead of at construction.
Otherwise there is a time window where the value is uninitialized and it's nicer if we can just avoid that.